### PR TITLE
Create Match page

### LINKED ISSUE #34: Create Proposals/accepts page
As a user
I want to have an overview about my proposals and my 'matches'
therefore an page displaying this is necessary.
Screenshot 2021-09-13 at 13 30 50

### PR DESCRIPTION
**Pull request by aronvaupel**
Implement most suggestions
Rerun npm i in an attempt to fix problems with heroku. PLS CHECK

closes #34

### CODE REVIEW

File: src/app/pages/MainPage/MainPage.tsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+            {mockupData.map((item) => (
+              <MainCard
+                imageSrc={item.src}
+                ratingValue={item.value}
+                ratingCondition={item.condition}
+                {...item}
+              />
+            ))}
+          </Carousel>
+        </section>{' '}
```
Comment (marwinburesch) [inline src/app/pages/MainPage/MainPage.tsx:59]:
```suggestion
        </section>
```

File: src/app/pages/MatchPage/MatchPage.tsx [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+  const handleClick = () => {
+    console.log('click');
+  };
+
+  return (
+    <div className={styles.wrapper}>
+      <Header />
+      <main className={styles.mainWrapper}>
+        <section className={styles.upper}>
+          <p>Swaps you proposed</p>
```
Comment (marwinburesch) [inline src/app/pages/MatchPage/MatchPage.tsx:42]: Reads like a `h2` to me

File: src/app/pages/MatchPage/MatchPage.tsx [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+        <section className={styles.lower}>
+          <p>Swaps both sides agreed to</p>
+          <Carousel
+            infiniteLoop={true}
+            showThumbs={false}
+            className={styles.carousel}
+          >
+            {mockupData.map((item) => (
+              <div>
+                {' '}
```
Comment (marwinburesch) [inline src/app/pages/MatchPage/MatchPage.tsx:71]:
```suggestion

```

File: src/app/pages/MainPage/MainPage.tsx [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+  const handleClick = () => {
+    console.log('click');
+  };
+
+  return (
+    <div className={styles.wrapper}>
+      <Header />
+      <main className={styles.mainWrapper}>
+        <section className={styles.upper}>
+          <p>Your offer</p>
```
Comment (marwinburesch) [inline src/app/pages/MainPage/MainPage.tsx:48]: `h2` here would be better here as well

File: src/app/pages/MainPage/MainPage.tsx [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+                ratingValue={item.value}
+                ratingCondition={item.condition}
+                {...item}
+              />
+            ))}
+          </Carousel>
+        </section>
+      </main>
+      <div className={styles.iconBackground}>
+        <IconButton iconType="swap" isActive={true} onClick={handleClick} />
```
Comment (marwinburesch) [inline src/app/pages/MainPage/MainPage.tsx:76]: You can leave the wrapping `div` out if you could pass `className` to this component

File: src/app/pages/MainPage/MainPage.tsx [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+              <MainCard
+                imageSrc={item.src}
+                ratingValue={item.value}
+                ratingCondition={item.condition}
+                {...item}
+              />
+            ))}
+          </Carousel>
+        </section>{' '}
+        <div className={styles.separationLine}></div>
```
Comment (marwinburesch) [inline src/app/pages/MainPage/MainPage.tsx:60]: A potential component :)

File: src/app/pages/MainPage/MainPage.tsx (hunk 5 above)
Comment (aronvaupel) [reply src/app/pages/MainPage/MainPage.tsx:76]: no

**Pull request merged**